lib/sles4sap/console_redirection.pm

```diff
@@ -139,6 +144,8 @@ sub connect_target_to_serial {
     );
     enter_cmd $ssh_cmd;
     handle_login_prompt($args{ssh_user});
+    # Create file with worker VM id on redirected machine.
+    assert_script_run("echo '$origin_id' > $redirection_file");
```

Comment:
I have one minor comment, do we have to use a file here, how about save the `machie-id` as a variable? :)

Reply:
But it is really a great/cool PR.

Reply:
I guess it's a personal approach. I look at the OpenQA variables rather as test settings so I prefer not to use them deep inside the code. OpenQA variable/setting is also changeable from outside of function so if there is a same setting name used somewhere completely else, value will be overwritten.